netmaster/objApi/apiController.go

```diff
@@ -737,38 +737,40 @@ func (ac *APIController) TenantCreate(tenant *contivModel.Tenant) error {
 		return err
 	}
 
-	// Create private network for the tenant
-	err = contivModel.CreateNetwork(&contivModel.Network{
-		Key:         tenant.TenantName + ":" + "private",
-		IsPublic:    false,
-		IsPrivate:   true,
-		Encap:       "vxlan",
-		PktTag:      1001,
-		Subnet:      "10.1.0.0/16",
-		Gateway:     "10.1.254.254",
-		NetworkName: "private",
-		TenantName:  tenant.TenantName,
-	})
-	if err != nil {
-		log.Errorf("Error creating privateNet for tenant: %+v. Err: %v", tenant, err)
-		return err
-	}
+	if tenant.TenantName == "default" {
```

Comment:
@mapuri: Just to give you some context, I'm working on removing auto creation of these networks completely(including from default tenant). I asked Divya to make a quick change to create these networks only for default tenant for now. She was running into problems with testing multiple tenants because of overlapping Vlan/Vxlan Ids for these auto created networks.